BrowserKit/Sources/ComponentLibrary/Buttons/LinkButtonViewModel.swift

```diff
@@ -4,6 +4,7 @@
 
 import UIKit
 import Foundation
+import Common
```

Comment:
remove?

Reply:
When I try to set the default value to 16 as you requested in the above comment, I receive an error that makes me import this line

Reply:
ah yes, you'll need it now for FXFontStyles, let's keep it thanks! :)